pkg/infra/log/file.go

```diff
@@ -193,12 +193,12 @@ func (w *FileLogWriter) DoRotate() error {
 		// close fd before rename
 		// Rename the file to its newfound home
 		if err = os.Rename(w.Filename, fname); err != nil {
-			return fmt.Errorf("Rotate: %s", err)
+			return fmt.Errorf("rotate: %s", err)
```

Comment:
Do we want the error be wrapped here?

Reply:
@papagian Not sure, I didn't really take it into account as part of this PR. I actually read a commentary in the meantime on how you don't _always_ want to wrap errors, in case you want to avoid leaking them to your callers. It would probably require some more thought to tell if we want to wrap here or not.

Reply:
ok, let's leave it as it used to be for now.